ee/vellum_ee/workflows/tests/test_display_meta.py

```diff
@@ -143,3 +143,53 @@ class Outputs(BaseWorkflow.Outputs):
     assert "result" in node_display.output_display
 
     assert "final_result" in display_meta.workflow_outputs
+
+
+def test_gather_event_display_context__final_output_node_workflow_output_id():
+    # GIVEN a workflow with a FinalOutputNode
+    files = {
+        "__init__.py": "",
+        "workflow.py": """\
+from vellum.workflows import BaseWorkflow
+from vellum.workflows.nodes import BaseNode
+from vellum.workflows.nodes.displayable import FinalOutputNode
+
+class DataNode(BaseNode):
+    class Outputs(BaseNode.Outputs):
+        value: str
+
+class MyFinalOutput(FinalOutputNode):
+    class Outputs(FinalOutputNode.Outputs):
+        pass
+
+class TestWorkflow(BaseWorkflow):
+    graph = DataNode >> MyFinalOutput
+    
+    class Outputs(BaseWorkflow.Outputs):
+        result = MyFinalOutput.Outputs.value
+""",
+    }
+
+    namespace = str(uuid4())
+    sys.meta_path.append(VirtualFileFinder(files, namespace))
+
+    # WHEN the workflow display context is gathered
+    display_meta = BaseWorkflowDisplay.gather_event_display_context(namespace)
+
+    # THEN the workflow display context should be created
+    assert display_meta is not None
+    assert isinstance(display_meta, WorkflowEventDisplayContext)
+
+    # AND the workflow_outputs should map to a workflow output variable ID
+    assert "result" in display_meta.workflow_outputs
+    workflow_output_id = display_meta.workflow_outputs["result"]
+
+    # AND the workflow output ID should NOT be the same as the node's output ID
+    node_displays = list(display_meta.node_displays.values())
+    final_output_node_display = [nd for nd in node_displays if "value" in nd.output_display][0]
+    node_output_id = final_output_node_display.output_display["value"]
+
+    assert workflow_output_id != node_output_id, (
+        f"Workflow output ID ({workflow_output_id}) should not be the same as "
+        f"node output ID ({node_output_id}). Workflow outputs need their own unique IDs."
+    )
```

Comment:
This test passes on main, so Idt it's right yet. Here's how to make it fail on main:
- Delete the DataNode
- `display_meta.node_displays.values()` should only have one element to compare to